src/quo2/components/onboarding/small_option_card/style.cljs

```diff
@@ -41,12 +41,10 @@
 
 (def touchable-overlay {:border-radius 16})
 
-(defn card-container
-  [main-variant?]
-  (when main-variant? {:height 343}))
-
 (defn card
   [main-variant?]
   {:background-color colors/white-opa-5
    :border-radius    16
    :height           (if main-variant? 335 56)})
+
+(def main-variant-extra-space {:height 8})
```

Comment:
When the card was pressed, the highlighted area was grater than expected, it was a bug I introduced while implementing this card in a previous PR, this change solves it.